scripts/main.js

```diff
@@ -0,0 +1,9 @@
+function toggleMenu() {
```

Comment:
**Kindly install `ESLint` in your linters project. You can have a copy of the setting files** [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js).
_After install, run your linters locally and ensure to no have issues after all working well without errors you can request another code review._